marquez_airflow/dag.py

```diff
@@ -217,15 +224,20 @@ def report_task(self,
                      airflow_dag_id=self.dag_id,
                      marquez_namespace=self.marquez_namespace)
 
-            marquez_jobrun_id = marquez_client.create_job_run(
-                step.name,
+            # TODO: Look into generating a uuid based on the DAG run_id
+            external_run_id = str(uuid4())
+
+            marquez_client.create_job_run(
+                namespace_name=self.marquez_namespace,
+                job_name=step.name,
+                run_id=external_run_id,
                 run_args=run_args,
                 nominal_start_time=start_time,
-                nominal_end_time=end_time).get('runId')
+                nominal_end_time=end_time)
 
-            if marquez_jobrun_id:
-                marquez_jobrun_ids.append(marquez_jobrun_id)
-                marquez_client.mark_job_run_as_started(marquez_jobrun_id)
+            if external_run_id:
+                marquez_jobrun_ids.append(external_run_id)
+                marquez_client.mark_job_run_as_started(external_run_id)
```

Comment:
we'd want to add the `at` parameter in the `mark_job_run_as_...` calls as well.

Reply:
The [client adds the transition time](https://github.com/MarquezProject/marquez-python/blob/main/marquez_client/client_wo.py#L193) if it's not provided

Reply:
Ah, I missed that. Sounds great.